tests/python_package_test/test_sklearn.py

```diff
@@ -82,7 +82,8 @@ def objective_ls(y_true, y_pred):
         X, y = load_boston(True)
         X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=42)
         gbm = lgb.LGBMRegressor(n_estimators=50, silent=True, objective=objective_ls)
-        gbm.fit(X_train, y_train, eval_set=[(X_test, y_test)], early_stopping_rounds=5, verbose=False)
+        gbm.fit(X_train, y_train, eval_set=[(X_test, y_test)], eval_metric='regression',
+                early_stopping_rounds=5, verbose=False)
```

Comment:
can we test these by only `metric`, therefore, we can check the backward compatible.